web/forms.py

```
@@ -1971,3 +1975,112 @@ def clean_title(self) -> str:
+            "floor_color",
+            "desk_color",
+            "chair_color",
+            "board_color",
+            "number_of_rows",
+            "desks_per_row",
+            "has_plants",
+            "has_windows",
+            "has_bookshelf",
+            "has_clock",
+            "has_carpet",
+        ]
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)